lib/rubygems.rb

```diff
@@ -1012,6 +1012,10 @@ def self.suffix_pattern
     @suffix_pattern ||= "{#{suffixes.join(',')}}"
   end
 
+  def self.suffix_regexp
```

Comment:
document and mark private?

Reply:
I'd like to keep it inline with the about `suffix_pattern` method, so I'll make both private?